qipr_approver/approver/forms/about_you_form.py

```diff
@@ -61,3 +61,17 @@ def __init__(self, user=User()):
                             'options': [item.name for item in Suffix.objects.all()],
                             'selected': [item.name for item in user.person.suffix.all()]}
 
+        self.last_login = {'name': 'last_login',
+                           'label': 'Last Login',
+                           'type': 'text',
+                           'value': user.person.last_login_time or ''}
```

Comment:
Fun fact, datetime is a bad input type to use:<input> elements with type="datetime" do not show as any datetime field/calendar in any of the major browsers, except Safari.

I am thinking we let django just give it to us as text and we handle it like such in html. What are your thoughts @PFWhite